### ISSUE TITLE
New version for Sandstorm

### ISSUE THREAD

**Issue by JamborJan** [jparyani/laverna-sandstorm#3]
Would it be possible to get the latest version packed for sandstorm? Especially tags and notebooks are useful features users are asking for. They are available in the latest version from the upstream repo.

Please let me know if I can support you and prepare a PR.

(FYI @germakoci)

**Comment by ocdtrekkie** [comment]
As a thought note, I'd say that if you can test the new version against Sandstorm, and prep a PR, that's probably an ideal way to not bog down the core Sandstorm devs with testing and deploying app updates. Even if there's things to fix that needs their help, it could save them some time.

**Comment by JamborJan** [comment]
Absolutely agree with you, that's why I asked. Will send a PR soon. Thx.

**Comment by jparyani** [comment]
I'd love a PR updating Laverna :)

Notebooks were intentionally removed at one point since they were causing problems (unfortunately I forget exactly what that problem was). I think the UI for it was somewhat broken, and it was hard/impossible to select notebooks. 

I'm all for tags, and I'm ok with adding support for notebooks back in as long as it's all working.

**Comment by germakoci** [comment]
would love to test this with my group and give feedback on what works, since the latest version really looks nice for collaborative work.

**Comment by JamborJan** [comment]
Hey guys, 

Short update. I was able to do some stuff, see https://github.com/JamborJan/laverna-sandstorm
- I introduced the `vagrant-spk` build process
- The build process automatically pulls in the latest upstream code which makes new versions super easy to build

It is still not working 100% out of the box but I think it's only cleanup work, the main points are:
- `make` doesn't take care about the sandstorm stuff in `.sandstorm/`. I need to find the place where this is configured
- Saving data doesn't yet work because ...
- ... I need to apply sandstorm specific changes like @jparyani has done it in his repo too
- many deprecated npm modules, need to check what happened there

If the weekend is as rainy as the weather forecast announced it I should have some time to do this. Otherwise it might take some more time.

Cheers
